src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -187,6 +187,8 @@ private static (T Result, string Output) UseTextWriter<T>(Encoding encoding, Fun
             bool shouldRunOnServer = true)
         {
             var arguments = new List<string>(argumentsSingle.Split(' '));
+            arguments.Add("/preferreduilang:en");
```

Comment:
It might be worth adding a comment here to the test that is validating that localization _does_ work, just so somebody isn't thinking we never test that.